clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -1686,7 +1686,7 @@ scheduler_impl_tmpl_t<RecordType, ReaderType>::open_readers(
         // Skip the auxiliary files.
         if (fname == DRMEMTRACE_MODULE_LIST_FILENAME ||
             fname == DRMEMTRACE_FUNCTION_LIST_FILENAME ||
-            fname == DRMEMTRACE_ENCODING_FILENAME)
+            fname == DRMEMTRACE_ENCODING_FILENAME || fname == DRMEMTRACE_V2P_FILENAME)
```

Comment:
Does this need to be skipped in other logic too? E.g.,

https://github.com/DynamoRIO/dynamorio/blob/73aae264aa7a41bf930b110572ea8f91add93d1c/clients/drcachesim/scheduler/scheduler_impl.cpp#L162

https://github.com/DynamoRIO/dynamorio/blob/73aae264aa7a41bf930b110572ea8f91add93d1c/clients/drcachesim/scheduler/scheduler_impl.cpp#L1687

Reply:
I have not seen issues with not having `DRMEMTRACE_V2P_FILENAME ` at line 162.
I believe that just creates a `default_file_reader_t` that is not used.
I don't see a downside adding it there too though, so done.

At line 1687, that's my change.